src/CalcManager/UnitConverter.h

```diff
@@ -228,7 +230,7 @@ namespace UnitConversionManager
         bool CheckLoad();
         double Convert(double value, ConversionData conversionData);
         std::vector<std::tuple<std::wstring, Unit>> CalculateSuggested();
-        void Reset();
+        void ResetCategoriesAndRatios();
```

Comment:
`IUnitConverter` declares all of its pure virtual functions as public.  In `UnitConverter`, some of those overriden functions are public and some are private.  While not technically an error according to the standard, it's at least strange.  There is an area above where most of the `IUnitConverter` overrides are publicly declared and grouped.  I'd appreciate if you moved the private declarations to that group and added the `override` specifier.